content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "About Arduino accounts for minors"
+---
+
+Arduino offers two types of accounts for minors:
+
+* [Junior accounts](#Junior-accounts). For users below 14 years old.
+* [Adult accounts with restrictions](#Adult-accounts). For users between 14 and 16 years old.
```

Comment:
Perhaps we should call them "regular" accounts instead of "adult".

This is how they're referred to in the upgrade prompt, and it makes sense since 14 year olds are not adults.

![image](https://user-images.githubusercontent.com/71350948/166651642-516f7871-2f62-435f-bcde-c22eca5b2d72.png)

Reply:
Yes, I agree, it is less confusing that way since the image calls it a "regular account". My only concern is that in Arduino's privacy policy, these accounts are referred to as "Adult account (for users between 14 and 16 years old" and this can create confusion.

Reply:
Updated [b55cbb9](https://github.com/arduino/help-center-content/pull/62/commits/b55cbb91f85b8f5c489bf267b539df5efaf7527a)

Reply:
@Renat0Ribeir0 Ah, then I see where you were coming from. But yeah on the whole I think it's a change for the better.